scripts/netContain/Dockerfile

```diff
@@ -17,30 +17,18 @@
 
 FROM ubuntu:16.04
```

Comment:
@unclejack, not related to this change, but when you are looking to change the Dockerfile, can you please take a look at https://github.com/contiv/netplugin/blob/master/install/v2plugin/Dockerfile and keep things similar if possible.